tools/convert_testcases/convert_testcase_helper.py

```diff
@@ -0,0 +1,411 @@
+from datetime import datetime
+import pytz
+import re
+
+# Define short names for various data types
+short_name_map = {
+    "string": "str",
+    "boolean": "bool",
+    "varbinary": "vbin",
+    "timestamp": "ts",
+    "timestamp_tz": "tstz",
+    "interval_year": "iyear",
+    "interval_day": "iday",
+    "interval": "iday",
+    "decimal": "dec",
+    "precision_timestamp": "pts",
+    "precision_timestamp_tz": "ptstz",
+    "fixedchar": "fchar",
+    "varchar": "vchar",
+    "fixedbinary": "fbin",
+}
+
+long_name_map = {v: k for k, v in short_name_map.items()}
+
+# Mapping of timezone abbreviations to pytz time zones
+timezone_abbr_map = {
+    "PST": "America/Los_Angeles",
+    "EST": "America/New_York",
+    "CST": "America/Chicago",
+    "MST": "America/Denver",
+    "HST": "Pacific/Honolulu",
+    "UTC": "UTC",
+}
+
+SQUOTE = "\u200B"
+DQUOTE = "&"
+
+# Map timezone abbreviations to valid pytz timezone names
```

Comment:
Note that this assumes daylight savings time is not in effect.  This shouldn't matter for our computation however.